.gitignore

```diff
@@ -116,3 +117,11 @@ dmypy.json
 
 # setuptools-scm
 version.py
+
+# Ape stuff
+tests/hardhat.config.js
+
+# NPM
+package.json
```

Comment:
It needs `package.json` for hardhat to run?

Reply:
No, I think @unparalleled-js just suggested adding a bunch of gitignore defaults for Node projects. Seems harmless to ignore but I can remove it if you want.

Reply:
yeah, harmless here was just curious